plugins/spf.js

```diff
@@ -222,17 +221,17 @@ exports.hook_mail = function (next, connection, params) {
 
 exports.log_result = function (connection, scope, host, mfrom, result, ip) {
     connection.loginfo(this, [
-        'identity=' + scope,
-        'ip=' + (ip ? ip : connection.remote.ip),
-        'domain="' + host + '"',
-        'mfrom=<' + mfrom + '>',
-        'result=' + result
+        `identity=${scope}`,
+        `ip=${ip ? ip : connection.remote.ip}`,
+        `domain="${host}"`,
+        `mfrom=<${mfrom}>`,
+        `result=${result}`
     ].join(' '));
```

Comment:
this join is also doing string concatenation. Which means this block could be collapsed to:
```js
connection.loginfo(this, `identity=${scope} ip=${ip ? ip : connection.remote.ip} domain="${host}" mfrom=<${mfrom}> result=${result}`
```

as well. Or to make for shorter lines, the ternaries could be extracted:

```js
const show_ip=ip ? ip : connection.remote.ip
connection.loginfo(this, `identity=${scope} ip=${show_ip} domain="${host}" mfrom=<${mfrom}> result=${result}`
```

Reply:
Sure. I'll make the changes and update PR.